### PR TITLE
Ansible: install the image service in the controller node

### PR DESCRIPTION
**Pull request by albertomurillo**

### CODE REVIEW

**Comment by albertomurillo** [comment]
Now that the image service is embedded into controller, the deployment script only creates the image service and endpoint and updates configuration.yaml accordingly.

File: _DeploymentAndDistroPackaging/ansible/roles/ciao-controller/defaults/main.yml [hunk 1]
```diff
@@ -56,24 +56,24 @@ ciao_openstack_services:
...
+  - service: glance
+    type: image
+    description: CIAO Image Service
 
 # CIAO endpoints to create in Keystone
 ciao_openstack_endpoints:
   - service: ciao
     type: compute
     public_url: https://{{ ciao_controller_fqdn }}:8774/v2.1/%(tenant_id)s
-    internal_url: https://{{ ciao_controller_fqdn }}:8774/v2.1/%(tenant_id)s
```
Comment (obedmr) [inline _DeploymentAndDistroPackaging/ansible/roles/ciao-controller/defaults/main.yml]: Is this something new in openstack playbooks? (to dont put the internal and admin urls)

Comment (albertomurillo) [inline _DeploymentAndDistroPackaging/ansible/roles/ciao-controller/defaults/main.yml]: internal and admin will be the same as public if not specified.
See _DeploymentAndDistroPackaging/ansible/roles/ciao-controller/tasks/endpoints.yml diff

**Comment by albertomurillo** [comment]
@tpepper @obedmr can you re-review this one?

File: _DeploymentAndDistroPackaging/ansible/roles/ciao-controller/defaults/main.yml (hunk 1 above)
Comment (obedmr) [inline _DeploymentAndDistroPackaging/ansible/roles/ciao-controller/defaults/main.yml]: cool

**Pull request merged**